libraries/protocol/include/golos/protocol/operations.hpp

```diff
@@ -60,6 +60,8 @@ namespace golos { namespace protocol {
                 reset_account_operation,
                 set_reset_account_operation,
                 delegate_vesting_shares_operation,
+                delegate_vesting_shares_with_interest_operation,
+                reject_vesting_shares_delegation_operation,
```

Comment:
you can't change older operations order, move this lines before virtual operations